### PR TITLE
[Doc Improvement][Sbs workflowbot v5][3069307]

### PR DESCRIPTION
**Pull request by v-ganr**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/46dfaee8713bd679c0758e66339f19944b6dbf30/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 85 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cbe22564-696a-41cf-bce5-fd75d96c91a4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/8b7bc8e7cd3dbb92a8b9d7b0e04a7b36e36f7994/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 85 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4bd08324-937e-407c-bd09-48161d2d75ec) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/c8b9d7585dec8b798c1c45944afb058375d6ea17/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 85 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e9c62399-3abe-4ad4-be17-4211deb6537f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/fe8fb3c7baf301bf37a6db92d0006a807cb3084c/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 85 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7a3732ea-6b1d-40ec-8466-b70aba445b66) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-gs-workflow-bot.yml [hunk 1]
```diff
@@ -110,53 +111,54 @@ items:
...
+
+     :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-add-app.png" alt-text="Screenshot shows the selction to add your workflow bot app.":::
     ### Take tour of source code
     Teams Toolkit provides components for building an app. After creating the project, you can view the project folders and files in the Explorer area of Visual Studio Code for JavaScript/TypeScript. 
     
-    :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-scaffolding2.png" alt-text="select add to add your workflow bot app":::
+    :::image type="content" source="./assets/images/sbs-workflow-bot/sbs-workflow-bot-scaffolding2.png" alt-text="Screenshot shows the selction to add your workflow bot app.":::
     
     
     The created bot is a normal TeamsFx project that contains following folders:
```
Comment (dooriya) [inline msteams-platform/sbs-gs-workflow-bot.yml:164]: The folder structure is out-of-date in the latest version workflow app.
Please refer to the latest [README.md]( https://github.com/OfficeDev/TeamsFx/tree/main/templates/scenarios/js/workflow#whats-included-in-the-template) to update it.

File: msteams-platform/sbs-gs-workflow-bot.yml [hunk 2]
```diff
@@ -196,20 +198,20 @@ items:
   durationInMinutes: 2
   content: |
     You can interact with the bot in Teams by sending a helloWorld command. After running this template, you get a response then select the **workflow bot** and **Enter**.
```
Comment (dooriya) [inline msteams-platform/sbs-gs-workflow-bot.yml:200]: Seems this sentence can be removed.

File: msteams-platform/sbs-gs-workflow-bot.yml [hunk 3]
```diff
@@ -23,7 +23,7 @@ items:
```
Comment (dooriya) [inline msteams-platform/sbs-gs-workflow-bot.yml:23]: The wording in the statement is confusing. It is unclear whether it refers to customizing the installation of the bot app or customizing the Adaptive Card response of the bot app. However, neither of these aspects are demonstrated in this tutorial.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b6095326c6bcd83d5fcc0c1412709ca62ebb5a10/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 85 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8a958b53-b53b-4cb6-90d8-ea7627e3082c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/854bb76f867ae50c590f7449b8e4f75905883045/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 85 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8e81296e-dc3e-478b-b479-c30e667a00ab) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b8959e474be9deceb404cc58b246cedb05ba8520/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 85 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b444dc04-a68f-47a7-af7d-9801c80b65da) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/2accd9d8e51434ca88fb78c439fb84a951c6d88d/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 85 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3e1c2f6b-6c04-484c-b590-493fe425844e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinx-at-msft2** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/a591b0a279d5da73aede0e971e2b0c5b58e0e77f/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 85 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a474a60c-4b43-4fcf-8812-87cea3508664) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/bd37922557eda7d99d8c732cc73e79c55a218a2a/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 85 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/87fa407b-3f70-4418-8c93-f08f1a4c2463) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/4408ab127962adfbefcd4072beb3250ce24e46c1/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 85 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e23d72c3-e315-427e-b61d-d31a32818bed) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/fb66a5d729b6b33506b50c70635d289d62eeefec/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9159d0c1-1d1a-4be1-8be0-6f2c31094192) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/1d5f4c56722b2a1bc8785826ff843ebe9661cc9c/msteams-platform%2Fsbs-gs-workflow-bot.yml) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/aaf024d371fd29bc2cc0950161bdeeed16f922de/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e98bba7f-740e-4fdc-8d69-98ef41242b11) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/4899ee0e0cb594933f57b1d9d7ba0148e50f2d5e/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cbb2eb72-592f-497a-b2a0-c949dbd5b7a3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/709f37e0b04f38d152986a1ee93af21bdc192761/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9e3cfadc-90e1-4675-b087-7ec712ce29e8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-gs-workflow-bot.yml (hunk 3 above)
Comment (v-bvishnu) [reply msteams-platform/sbs-gs-workflow-bot.yml:22]: Updated.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/1480eb0db31de5bd729a95cde0f594e30fbb0f1c/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fbda6824-0762-4a8d-b567-c2b1ffc1decc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-gs-workflow-bot.yml (hunk 2 above)
Comment (dooriya) [reply msteams-platform/sbs-gs-workflow-bot.yml]: I mean remove "After running this template, you get a response then select the **workflow bot** and **Enter**.", and still keep "You can interact with the bot in Teams by sending a **helloWorld** command."

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/99bb2fa3cd6326e19844f002b06c5dd76b75079d/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ea47af94-b7f0-4ccc-94dd-7c585d1eef2e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/1a89e3ac04a040dd66dfba73090f5d841cbceeed/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/33e6bb02-ead2-40fb-85b0-905aa4da0c05) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by summzhan — APPROVED** [review]

**Review by v-habibaumme — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/d65edbb86e7bf5bf96f900016e28eb9dc42c7875/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/18ab557b-2b62-4729-9544-0e3ea3c25b17) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-workflow-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/445178012cd4a189deffbb7e7e192289afb70cda/msteams-platform%2Fsbs-gs-workflow-bot.yml) | 89 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f3c99c3e-adbc-4aac-9e9f-837e800e1297) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**